lib/matplotlib/tests/test_widgets.py

```diff
@@ -1033,8 +995,7 @@ def test_TextBox(ax, toolbar):
 
 
 @image_comparison(['check_radio_buttons.png'], style='mpl20', remove_text=True)
-def test_check_radio_buttons_image():
-    ax = get_ax()
+def test_check_radio_buttons_image(ax):
```

Comment:
```suggestion
def test_check_radio_buttons_image():
    ax = get_ax()
```

The use of this fixture seems to be fighting with our other fixture?